commands/lmpop.md

```diff
@@ -0,0 +1,34 @@
+Pops one ore more elements from the first non-empty list key from the list of provided key names.
+
+Similar other commands have limitations:
```

Comment:
```suggestion
LMPOP and BLMPOP deprecate the following commands:
```

Reply:
@enjoy-binbin removed the deprecation mention after we agreed we're not deprecating the simple commands. Please review the suggestion above.

Reply:
Took it! thanks for the all effort